### PR TITLE
v1.34 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
 
+## 1.34.0 Release
+- Release Date: August 16, 2020
+- Server Versions Supported: Server v5.19+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.19 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.9 has ended and upgrading to server ESR v5.19 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app. See [this blog post](https://mattermost.com/blog/support-for-esr-5-9-has-ended/) for more details.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
```
Comment (amyblais) [inline CHANGELOG.md:11]: Breaking Change?
We've replaced auto-scrolling to the new message line with a "More Messages" button. If the new message line is the  next viewable post, or it is hidden behind the top bar, then in this case we do automatically scroll.

**Comment by lindalumitchell** [comment]
FYI: Testing building the apps from this PR since we don't need test builds for it and so this shouldn't be in the way of actual testing. Checking on the new webhook to have Fastlane post in the new QA: Builds channel.

File: CHANGELOG.md (hunk 1 above)
Comment (enahum) [reply CHANGELOG.md:11]: I don’t consider this breaking

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+### Improvements
+ - Removed auto-scrolling to the new message line on channel load and added a "More Messages" button when there are unread posts.
+ - Improved screen styling for iOS Settings, Profile, Channel Info, "+" button for DMs and channels, Create Channel, and other user profile pages.
+ - Added the ability to view users' first and last name in the profile view.
+ 
+### Bug Fixes
+
+#### All apps
+ - Fixed an issue with More Messages button inconsistencies when opening the app from a push notification.
```
Comment (enahum) [inline CHANGELOG.md:25]: Remove, this is from a new feature

**Comment by amyblais** [comment]
Found a few other improvements,

**Comment by enahum** [comment]
I think this one https://github.com/mattermost/mattermost-mobile/pull/4373 is huge and massive props to @migbot as messages are no longer entering and remaining behind the keyboard.

What about showing the error in the sso screen instead of a long running spinner? https://github.com/mattermost/mattermost-mobile/pull/4609

there is also this one https://github.com/mattermost/mattermost-mobile/pull/4647 filed by you @amyblais

**Comment by amyblais** [comment]
https://github.com/mattermost/mattermost-mobile/pull/4373 was already in the changelog.
Added https://github.com/mattermost/mattermost-mobile/pull/4609 (originally thought it was a temporary fix).
From the PR description I thought https://github.com/mattermost/mattermost-mobile/pull/4647 was a regression, but I added it.

**Comment by amyblais** [comment]
@enahum This can be merged

**Pull request merged**